py/qqa/qa/noisecorr.py

```diff
@@ -0,0 +1,102 @@
+from .base import QA
+import glob
+import os
+import collections
+
+import numpy as np
+import fitsio
+
+
+from astropy.table import Table
+
+import desiutil.log
+from desispec.preproc import _overscan
+
+def _fix_amp_names(hdr):
+    '''In-place fix of header `hdr` amp names 1-4 to A-D if needed.'''
+    #- Assume that if any are right, all are right
+    if 'DATASECA' in hdr:
+        return
+
+    log = desiutil.log.get_logger()
+    log.debug('Correcting AMP 1-4 to A-D for night {} expid {}'.format(
+        hdr['NIGHT'], hdr['EXPID']))
+
+    for prefix in [
+        'GAIN', 'RDNOISE', 'PRESEC', 'PRRSEC', 'DATASEC', 'TRIMSEC', 'BIASSEC',
+        'ORSEC', 'CCDSEC', 'DETSEC', 'AMPSEC', 'OBSRDN', 'OVERSCN'
+        ]:
+        for ampnum, ampname in [('1','A'), ('2','B'), ('3','C'), ('4','D')]:
+            if prefix+ampnum in hdr:
+                hdr[prefix+ampname] = hdr[prefix+ampnum]
+                hdr.delete(prefix+ampnum)
```

Comment:
Blatant copy-and-paste from `qqa.qa.amp._fix_amp_names`.  Let's `from .amp import _fix_amp_names` (possibly upgrading it to be a non _underscore hidden function) instead of replicating here.